<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QPushButton" name="ButtonAddComplex">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="ButtonRemoveComplex">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="ButtonShowCurve">
      <property name="text">
       <string>Show Curve</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QTableWidget" name="tableProteinComplex"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="labelProteinComplex">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Protein Complex</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QListWidget" name="ListFile"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="ButtonGroup1">
      <property name="text">
       <string>Set as G1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QTableView" name="tableProtein1"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="labelDatabase">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Database List</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="labelThermalShift2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Group2 Thermal Shift Curve</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QTableView" name="tableProtein2"/>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="labelThermalShift1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Group1 Thermal Shift Curve</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelFileList">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>File List</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="labelProteinTable1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Group1 Protein Table</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="labelProteinTable2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Group2 Protein Table</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="ButtonGroup2">
      <property name="text">
       <string>Set as G2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="ButtonClearFileList">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" rowspan="4">
     <widget class="QGraphicsView" name="GraphicThermShift1"/>
    </item>
    <item row="4" column="4" rowspan="4">
     <widget class="QGraphicsView" name="GraphicThermShift2"/>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QListWidget" name="ListDatabase"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="ButtonDatabaseRemove">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="ButtonClearDatabase">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="ButtonDatabaseConfirm">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionPreprocessing"/>
    <addaction name="actionProteomics"/>
    <addaction name="actionDatabase"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCheck_update"/>
    <addaction name="actionCitation"/>
    <addaction name="actionCheck_liences"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionCalcROC"/>
    <addaction name="action_CETSA"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>img/open.png</normaloff>img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>img/save.png</normaloff>img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>img/quit.png</normaloff>img/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionProteomics">
   <property name="text">
    <string>Load proteomics...</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="text">
    <string>Load PPI database...</string>
   </property>
  </action>
  <action name="action_CETSA">
   <property name="text">
    <string>Single protein analysis (TSA) ...</string>
   </property>
  </action>
  <action name="actionCalcPval">
   <property name="text">
    <string>Calculate p-values of protein complex ...</string>
   </property>
  </action>
  <action name="actionCalcROC">
   <property name="text">
    <string>Protein pair ROC analysis ...</string>
   </property>
  </action>
  <action name="actionCheck_update">
   <property name="text">
    <string>Check update ...</string>
   </property>
  </action>
  <action name="actionCitation">
   <property name="text">
    <string>Check citation ...</string>
   </property>
  </action>
  <action name="actionCheck_liences">
   <property name="text">
    <string>Check licence</string>
   </property>
  </action>
  <action name="actionPreprocessing">
   <property name="text">
    <string>Preprocessing...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
